build.sbt

```diff
@@ -67,7 +67,15 @@ ThisBuild / githubWorkflowGeneratedCI += WorkflowJob(
   name = "Scalafix codegen files",
   steps = List(
     WorkflowStep.Checkout,
-    WorkflowStep.Sbt(commands = List("reload plugins", "scalafixAll --check"))),
+    WorkflowStep.Sbt(commands = List("reload plugins", "scalafixAll --check", "test"))),
+  scalas = List(Scala212)
+)
+
+ThisBuild / githubWorkflowGeneratedCI += WorkflowJob(
+  id = "test-codegen",
+  name = "Tests for codegen",
+  steps =
+    List(WorkflowStep.Checkout, WorkflowStep.Sbt(commands = List("reload plugins", "test"))),
```

Comment:
It looks like the tests are being run both in this job and the scalafix job.

Btw, why separate it into two jobs? It seems reasonable to run the tests right after the scalafix check. Conceptually we have 10 sub-projects in this repo: root{JVM,JS,Native} x {2.12,2.13,3} + meta-build. So one CI job for each seems right.

Reply:
The double test was a typo. Sorry about that. I can roll it into one job though. I don't have strong feelings about it.